lib/helpers.dart

```diff
@@ -5,14 +5,27 @@ import 'package:stingray/history.dart';
 import 'package:stingray/model/item.dart';
 
 void handleShare(int id) {
```

Comment:
Dart does support method overloading so you need to delete this obsolete method, it looks like the only place that uses `handleShare` is updated to use the new function, so it should be safe for you to delete this whole thing :)